### PR TITLE
Packaged issue fixes from UXD team

### LINKED ISSUE #263: New blog posts are hidding inside 'blog' section giving no visibility
For the casual visitor, new blog posts are not featured in the website, a sidebar on the front page containing latest posts could be a good addition.

### LINKED ISSUE #380: html Blockquotes are not differentiate from the rest of the content
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug
/kind enhancement

**What happened**: When there is a markdown note indicated with '>' an html blockquote is generated, the css class doesn't distinguish that blockquote from the rest of the text, should be shadowed or boxed to see that there is a blockquote.

**What you expected to happen**: Get the text into the blockquote differentiate from the rest of the text.

**Anything else we need to know?**:

URL where the problem can be found: https://5daec8f20378a30008ccd782--kubevirtio.netlify.com/2019/kubevirt_k8s_crio_from_scratch_installing_kubevirt

### LINKED ISSUE #540: Spacer after/before code blocks
Code blocks are pretty close to paragraphs, this needs more tuning

Ref: [KNIECO-2271](https://issues.redhat.com/browse/KNIECO-2271)

### LINKED ISSUE #599: Beautify action buttons
/kind enhancement

We have a lovely logo and color theme yet action buttons colored orange .  They shout warning! and danger!

Can we get an update to them?

### PR DESCRIPTION
**Pull request by mazzystr**
**What this PR does / why we need it**:
mazzystr fork branch uxd will contain all issue fixes from UXD team so we can release as a single package

Fixes #263 
Fixes #380  
Fixes #540 
Fixes #599

### CODE REVIEW

**Comment by mazzystr** [comment]
@codificat @fabiand  Please review the following...
* Front page ... boundary borders loosened up
* Front page ...  Panel with recent blog posts added
* Front/Community page ... Try It! buttons modernized
* Blogs/Labs page ... left panel font size and position should match

**Comment by dankenigsberg** [comment]
> @codificat @fabiand Please review the following...
> 
>     * Front page ... boundary borders loosened up
> 
>     * Front page ...  Panel with recent blog posts added
> 
>     * Front/Community page ... Try It! buttons modernized
> 
>     * Blogs/Labs page ... left panel font size and position should match

@mazzystr can you put this can of explanation at the to description of the PR. How would you recommend to a layperson like myself to review the change?

**Review by codificat — CHANGES_REQUESTED** [review]
Only some minor things to update.

There are also a couple of design questions/comments, like the white gap above the main title in the labs and the left pane's lack of margin/padding on that page, but I think these are better addressed as minor follow-ups, so we don't delay this one any longer.

File: index.html [hunk 1]
```diff
@@ -2,72 +2,40 @@
...
+  <p class="kv-what__text">Support development of new microservice applications in containers that interact with existing virtualized applications.</p>
 </section>
+
+<section class="kv-recent-posts">
+  <h2>Recent posts</h2>
+  {% for post in site.posts limit:5 %}
+  <article class="kv-blog__snippet">
+    <a href="{{ post.url }}" class="kv-blog__snippet__link">
+      <h4 class="">{{ post.title }}</h4>
+      <time datetime="1999-12-23" class="">{{ post.date | date: "%a, %-d/%m/%y" }}</time>
```
[the remark this answers is not in the document]
Comment (mazzystr) [reply index.html:37]: I don't understand the commit or comment.  Let's issue this and move fwd.

**Comment by codificat** [comment]
Blocker issues were addressed, so in the interest of moving this bulk update forward:

/lgtm
/approve

There are some issues to be addressed here, but we'll open separate/specific issues/PRs to review them, including:
- date formatting in the new /Recent posts/ sidebar in the home page
- margins/padding of the left side bar and the main content area in labs and blogs
- block spacing has been fixed, but it might be that some existing content had workarounds for that (e.g. `<br>` lines) that should be cleaned up

**Pull request merged**